keel-core-test/src/main/kotlin/com/netflix/spinnaker/keel/persistence/DeliveryConfigRepositoryTests.kt

```diff
@@ -575,6 +578,41 @@ abstract class DeliveryConfigRepositoryTests<T : DeliveryConfigRepository, R : R
           }
         }
       }
+
+      context("notifications") {
+        before {
+          repository.store(deliveryConfig.copy(
+            environments = setOf(
+              Environment(
+                name = "prod",
+                resources = setOf(
+                  resource(kind = parseKind("ec2/cluster@v1")),
+                  resource(kind = parseKind("ec2/security-group@v1"))
+                ),
+                notifications = setOf(NotificationConfig(
+                  type = NotificationType.slack,
+                  address = "test",
+                  frequency = NotificationFrequency.verbose
+                ),
+                  NotificationConfig(
+                    type = NotificationType.email,
+                    address = "test",
+                    frequency = NotificationFrequency.quiet
+                  ))
+              ),
+            )
+          ))
+        }
+        test ("get stored notifications") {
+          expectThat(repository.environmentNotifications(deliveryConfig.name, "prod")?.size)
+            .isEqualTo(2)
+        }
+
+        test ("environment without notifications will return an empty set") {
+          expectThat(repository.environmentNotifications(deliveryConfig.name, "staging"))
+            .isNull()
```

Comment:
As mentioned above, this should be `.isEmpty()` so it will match the test description. 🙂